trust-store-trust-policy-specification.md

```diff
@@ -158,23 +184,19 @@ Here is high level uml diagram for signature evaluation:
 
 ## FAQ
 
-**Q1.** Can I scope trust policy by the registry, repository, or namespace?
-
-No, we don't support scoping of trust policy.
-
-**Q2.** How should multiple signatures requirements be represented in the trust policy?
+**Q1.** How should multiple signatures requirements be represented in the trust policy?
```

Comment:
To avoid future churn to these headers, maybe just make them bullet points, or numbered points?

Reply:
For formatting question and answer bullet or number points don't work because of indentation issue. Removed question number to avoid any future churn.